### PR TITLE
Setup unit testing.

### PR DESCRIPTION
**Pull request by samuelOyegbemi**
#### What does this PR do?
- Setup unit testing
- Added response helper

#### Description of Task proposed in this pull request?
- added first unit test in `test/index.spec.js`
- added `ok`, `success`, `created`, `error`, `notFound`, `badRequest` and `unAuthorize` methods to the response helper

#### How should this be manually tested (Quality Assurance)?
- cd into the project root directory
- Prepare your environment by following instruction in the README.md file
- Run `npm test`

#### What are the relevant pivotal tracker stories?
- [172622757](https://www.pivotaltracker.com/story/show/172622757)

#### Any background context you want to add (Operations Impact)?
- Run `npm install` to update dependencies

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,24 +24,19 @@ app.use(
...
-    code: 404,
-  });
+app.use((error, req, res, next) => {
+  if (res.headersSent) return next(error);
+  return response.error(res, error.status || 500, error, error.message || 'Internal Server Error');
 });
 
+app.use('*', (req, res) => response.notFound(res));
+
 app.use(helmet());
```
Comment (tolulope-od) [inline index.js]: Great work here Samuel, however, can we use helmet as the first middleware? It helps add the headers earlier in the application

**Review by tolulope-od — COMMENTED** [review]
Great work here! Please review the comment I left.

**Review by chuxmykel — APPROVED** [review]
Great job man. LGTM!

File: index.js (hunk 1 above)
Comment (samuelOyegbemi) [reply index.js]: Done

**Review by chuxmykel — APPROVED** [review]

**Review by tolulope-od — APPROVED** [review]
LGTM

**Pull request merged**